tests/py_enum_overload.py

```diff
@@ -0,0 +1,8 @@
+def test_enum1(nimobject):
```

Comment:
It is beneficial to add the tests to existing `pyfromnim` tests instead of creating new files. That's because pyfromnim is tested in both python-originated environment (when nim is python module) and python-imported environment (when nim is executable).